lib/search_config.rb

```diff
@@ -22,63 +35,16 @@ def index_names
     content_index_names + auxiliary_index_names
   end
 
-  def content_index_names
-    elasticsearch["content_index_names"] || []
-  end
-
-  def auxiliary_index_names
-    elasticsearch["auxiliary_index_names"] || []
```

Comment:
Should have mentioned that. There is no valid use case for `content_index_names` and `auxiliary_index_names` being nil/empty. They are either set by elasticsearch.yml or the test helpers.